openapi.yml

```diff
@@ -896,8 +896,18 @@ components:
               # description: An alphabet or other notation used to represent a
               #   language or other symbolic system of the descriptive element value.
     DOI:
-      type: string
       description: Digital Object Identifier (https://www.doi.org)
+      oneOf:
+        - $ref: '#/components/schemas/DOI_pattern'
+        - $ref: '#/components/schemas/DOI_exceptions'
+    DOI_exceptions:
+      type: string
+      description: pre-existing Digital Object Identifiers (https://www.doi.org) not matching the pattern
+      pattern: '^10\.(25740\/(VA90-CT15|syxa-m256|12qf-5243|65j8-6114)|25936\/629T-BX79)$'
```

Comment:
Would doing this as an `enum` instead of a `pattern` be clearer?

Reply:
good idea.  I will change.

Reply:
answer:  DOI is NOT case sensitive, so pattern is better.